server/scripts/fetchdata/ThronesDbImageSource.js

```diff
@@ -0,0 +1,21 @@
+/*eslint no-console:0 */
+
+const request = require('request');
+const fs = require('fs');
+
+class ThronesDbImageSource {
+    fetchImage(card, imagePath) {
+        let url = `https://thronesdb.com/bundles/cards/${card.code}.png`;
```

Comment:
I see why youve done this but this means we're no longer using the imageUrl property of the card, as long as we're ok with that

Reply:
The `imagesrc` property used before was only ever being used in fetchdata, and really had no use outside of there (since it refers to a ThronesDB specific path). Since it's only available in the ThronesDB API response and not in the JSON data, had to get rid of its usage so that you could mix card data sources (JSON) with image data sources (ThronesDB)

Reply:
Yeah that's what i meant by i see why you've done it. It means that in the unlikely event thronesdb has a reorg it will break, but it's a reasonable risk to accept